### ISSUE TITLE
Documentation of race formats

### ISSUE THREAD

**Issue by ErikBorra** [RotorHazard/RotorHazard#360]
There currently is no documentation of the various race formats

**Comment by HazardCreative** [comment]
Is this section missing something important?
https://github.com/RotorHazard/RotorHazard/blob/master/doc/User%20Guide.md#race-format

**Comment by ErikBorra** [comment]
Yes. The explanation of the different formats in the drop down menu, such as team race etc.

**Comment by HazardCreative** [comment]
The default formats are arrangements of the various format settings, and they can be modified and deleted by the end user. However, I'll reconsider how to handle this moving forward. I've just redone all of the win condition code in #339 and some of the new conditions are less obvious.

**Comment by ErikBorra** [comment]
Thanks for the feedback! Can I reliable use master/livetime-settings tomorrow, or should I use #339?

**Comment by HazardCreative** [comment]
It's not a stable release so I hesitate to use the word reliable, but there are no known bugs at the moment. If you're using LiveTime, you won't notice the changes in #339. If not, you won't notice the changes in #317. Or if you're pretty comfortable with git, you can merge both locally and try them together; they shouldn't overlap.

**Comment by ErikBorra** [comment]
Let me rephrase the question. Is the current master reliably detecting win conditions, or should I use #339?

**Comment by HazardCreative** [comment]
There are no known issues with times collected or reported on the current master.

The work in #339 extends and improves the announcement of a winner. Current master will only announce the winner of most-laps-wins or first-to-X-laps conditions. #339 resolves some edge cases with result sorting and display of position ranking, especially when using fastest-lap or 3-consecutive-lap conditions. Finally, #339 dramatically improves win conditions with respect to team racing and cleans up code so it's more easy to read and extend in the future.

**Comment by HazardCreative** [comment]
Closing with the merge of #339.
